docs/index.rst

```diff
@@ -1,4 +1,4 @@
-.. include:: README.rst
+.. include:: ../README.rst
```

Comment:
@imujjwal96 hyper link like this `javadoc <./javadoc>`_  and also add java build pack to Heroku javadoc is not generating